regcomp.h

```diff
@@ -226,6 +267,8 @@ struct regnode_charclass_posixl {
  * allocated in terms of multiples of a single-argument regnode.  SSC nodes can
  * have a pointer field because there is no alignment issue, and because it is
  * set to NULL after construction, before any cloning of the pattern */
+/* NOTE - this "regnode" is not used in the program itself, so you do not need
+ * to worry about member alignment issues here */
```

Comment:
This is already said in part in the preceding comment. It might be clearer to replace these extra lines with a slight augmentation of what precedes:
```
  [...] Also, there is no alignment issue,
  * because this regnode is declared or allocated as a complete unit and never
  * used in the program itself, to the compiler takes care of alignment. [...]
```

Reply:
I decided to rework that paragraph entirely as it was too dense.